### DISCUSSION TITLE
Implementation of TEBD routine

### DISCUSSION THREAD

**Discussion by ngm29** [Part-II-Computational-Physics/.github#98] [Q&A] [answered]
I am doing my project on TEBD for quantum spin chains, and so far, I have relied on the implementation of the algorithm by Glen Evenly on the website referenced in the project description. Am I expected to write my own implementation, or is it ok to use this one (with an appropriate discussion in the report)?

**Comment by AustenLamacraft** [comment, marked as answer]
It's OK to use this one and see what you can calculate: try some different Hamiltonians, real and imaginary time evolution, expectation values of observables...
